test/util/exportToNDJson.test.js

```diff
@@ -59,33 +61,42 @@ describe('check export logic', () => {
   });
 
   describe('exportToNDJson', () => {
+    beforeEach(async () => {
+      fs.rmSync('./tmp/123456', { recursive: true, force: true });
+    });
     test('Expect folder created and export successful when _type parameter is retrieved from request', async () => {
-      await exportToNDJson(clientId, mockType);
+      await exportToNDJson({ clientEntry: clientId, types: mockType });
       expect(fs.existsSync(expectedFileName)).toBe(true);
     });
     test('Expect folder created and export successful when _type parameter is not present in request', async () => {
-      await exportToNDJson(clientId);
+      await exportToNDJson({ clientEntry: clientId });
       expect(fs.existsSync(expectedFileName)).toBe(true);
     });
 
-    test('Expect folder created and export successful when _typeFilter  parameter is retrieved from request', async () => {
-      await exportToNDJson(clientId, mockType, mockTypeFilter);
+    test('Expect folder created and export successful when _typeFilter parameter is retrieved from request', async () => {
+      await exportToNDJson({ clientEntry: clientId, type: mockType, typeFilter: mockTypeFilter });
       expect(fs.existsSync(expectedFileName)).toBe(true);
     });
-    test('Expect folder created and export successful when _typeFilter  parameter is retrieved from request', async () => {
-      await exportToNDJson(clientId, mockType, complexMockTypeFilter);
+    test('Expect folder created and export successful when complex _typeFilter parameter is retrieved from request', async () => {
+      await exportToNDJson({ clientEntry: clientId, type: mockType, typeFilter: complexMockTypeFilter });
       expect(fs.existsSync(expectedFileName)).toBe(true);
       expect(fs.existsSync(expectedFileNameEncounter)).toBe(true);
       expect(fs.existsSync(expectedFileNameServiceRequest)).toBe(true);
     });
     test('Expect folder created and export to fail when _typeFilter parameter is retrieved from request and contains an invalid param', async () => {
-      await exportToNDJson(clientId, mockType, typeFilterWithInvalidType);
+      await exportToNDJson({ clientEntry: clientId, type: mockType, typeFilter: typeFilterWithInvalidType });
+      expect(fs.existsSync('./tmp/123456')).toBe(true);
       expect(fs.existsSync(expectedFileNameInvalidType)).toBe(false);
     });
-    test('Expect folder created and export to fail when _typeFilter  parameter is retrieved from request but the value set is invalid', async () => {
-      await exportToNDJson(clientId, mockType, typeFilterWOValueSet);
+    test('Expect export to fail when _typeFilter parameter is retrieved from request but the value set is invalid', async () => {
+      await exportToNDJson({ clientEntry: clientId, type: mockType, typeFilter: typeFilterWOValueSet });
+      // TODO: this currently fails and does not create a folder (throws error instead). Do we want a more graceful response?
```

Comment:
Ah I see I am assuming we want the same functionality as the test above it? Is it a quick fix or should this be a backlog task?

Reply:
I'm not actually sure we want a folder to be created in either case. Maybe since we're tagging @hossenlopp in things, we can get his opinion on this one too? :)

Reply:
Looking into this further, the invalid type in a type filter is checked in the export service before this function is called, so I don't think it really matters if it has appropriate checking in this function. Will plan on removing this TODO and adding a quick comment instead.